### ISSUE TITLE
Notes that contain a transcluded section of another note do not export correctly

### ISSUE THREAD

**Issue by two235** [mvdkwast/obsidian-copy-as-html#22]
For Example, exporting a note containing this:

```
### First Project
![[NoteID1#section10]]

### Second Project
![[NoteID2#section14]]

### Third Project
![[NoteID3#section5]]
```

Results in this:

```
First Project
NoteID1 > section10
Second Project
NoteID2 > section14
Third Project
NoteID3 > section5
```

Instead of the transcluded content

Transcluding the full note works correctly.

**Comment by RaffaeleBianc0** [comment]
+1 on this!
I am heavily relying on Daily Notes for a lot of my contents, using H1 headers, and linking to those H1 headers from other notes using the ![[Note#header]] format, to build up significant content with almost zero effort.
Actually I am forced to
- make a copy of the note I need to export, to a temporary note
- copy-and-paste all the transcluded content from my Daily Notes to the temporary note (replacing the links with actual content)
- export the temporary note to HTML
- delete the temporary note

**Comment by mvdkwast** [comment]
Resolved in 0.4.1
